jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/BuildImageMojo.java

```diff
@@ -233,24 +233,31 @@ void buildImage(BuildImageSteps buildImageSteps) throws MojoExecutionException {
   }
 
   /** Checks validity of plugin parameters. */
-  private void validateParameters() {
+  private void validateParameters() throws MojoFailureException {
+    boolean shouldFail = false;
+
     // 'tag' must not contain backslashes.
     if (tag.indexOf('/') >= 0) {
       getLog().error("'tag' cannot contain backslashes");
+      shouldFail = true;
```

Comment:
this tag check can probably just go at the end, and we can remove the shouldFail flag.